src/linkcommand.js

```diff
@@ -49,12 +70,34 @@ export default class LinkCommand extends Command {
 	 *
 	 * When the selection is collapsed and inside the text with the `linkHref` attribute, the attribute value will be updated.
 	 *
+	 * # Decorators and attribute management
+	 *
+	 * This command has an optional argument, which applies or removes model attributes brought by
+	 * {@link module:link/utils~ManualDecorator manual decorators}. Model attribute names correspond to
+	 * decorator {@link module:link/utils~ManualDecorator#id ids} and follow the incremental pattern:
+	 * `'linkManualDecorator0'`, `'linkManualDecorator1'`, `'linkManualDecorator2'`, etc..
+	 *
+	 * To learn more about link decorators, check out the {@link module:link/link~LinkConfig#decorators `config.link.decorators`}
+	 * documentation.
+	 *
 	 * @fires execute
 	 * @param {String} href Link destination.
+	 * @param {Object} [manualDecoratorIds={}] The information about manual decorator attributes to be applied or removed upon execution.
 	 */
-	execute( href ) {
+	execute( href, manualDecoratorIds = {} ) {
```

Comment:
How to add new decorator attributes and how to remove them (what to pass in `manualDecoratorIds` in both cases)?  This needs an example here.